tests/modules/base/application/patterns/unit/test_app_service.py

```diff
@@ -26,26 +25,3 @@ def info(self):
     app_service = NoInfoAppService()
```

Comment:
def should_*****() ?????????

Reply:
I don't know if understand it, tests without class I see good to keep them starting with `test_`

Reply:
my bad...  I only reviewed one part and I got confused. Your code is ok :)